src/main/resources/db/migration/V2.2__emails_incrice_langth.sql

```diff
@@ -0,0 +1,27 @@
+-- alter CAMPAIGNS
+
+ALTER TABLE CAMPAIGNS ADD COLUMN NEW_BODY VARCHAR(32672) NOT NULL default '';
+UPDATE CAMPAIGNS SET NEW_BODY=BODY;
+ALTER TABLE CAMPAIGNS DROP COLUMN BODY;
+RENAME COLUMN CAMPAIGNS.NEW_BODY TO BODY;
```

Comment:
Why not just do:
```
ALTER TABLE CAMPAIGNS MODIFY BODY VARCHAR(32672);
```

https://stackoverflow.com/questions/1279568/how-can-i-modify-the-size-of-column-in-a-mysql-table